operations/boot_orchestration/BOS_Workflows.md

```diff
@@ -1,36 +1,361 @@
 # BOS Workflows
 
-The following workflows present a high-level overview of common Boot Orchestration Service \(BOS\) operations. These workflows depict how services interact with each other when booting, configuring, or shutting down nodes. They also help provide a quicker and deeper understanding of how the system functions.
+The following workflows present a high-level overview of common Boot Orchestration Service \(BOS\) operations.
+These workflows depict how services interact with each other when booting, configuring, or shutting down nodes. They also help provide a quicker and deeper understanding of how the system functions.
+
+## BOS V2 Workflows
+
+The following workflows are included in this section:
+
+* [Boot Nodes](#v2-boot-nodes)
+* [Reboot Nodes](#v2-reboot-nodes)
+* [Power Off Nodes](#v2-power-off-nodes)
+
+### V2 Boot Nodes
+
+**Use Case:** Administrator powers on and configures select compute nodes.
+
+**Components:** This workflow is based on the interaction of the BOS with other services during the boot process:
+
+Mentioned in this workflow:
+
+* Boot Orchestration Service \(BOS\) is responsible for booting, configuring, and shutting down collections of nodes. The Boot Orchestration Service has the following components:
+  * Boot Orchestration Session Template is a collection of one or more boot set objects. A boot set defines a collection of nodes and the information about the boot artifacts and parameters.
+  * BOS sessions provide a way to apply a template across a group of nodes and monitor the progress of those nodes as they move toward their desired state.
+  * BOS Operators interact with other services to perform actions on nodes, moving them toward their desired state.
+* Cray Advanced Platform and Monitoring Control \(CAPMC\) service provides system-level power control for nodes in the system.
+CAPMC interfaces directly with the Redfish APIs to the controller infrastructure to effect power and environmental changes on the system.
+* Hardware State Manager \(HSM\) tracks the state of each node and their group and role associations.
+* Boot Script Service \(BSS\) stores per-node information about iPXE boot script. Nodes consult BSS for boot artifacts \(kernel, `initrd`, image root\) and boot parameters when nodes boot or reboot.
+* The Simple Storage Service \(Ceph S3\) is an artifact repository that stores boot artifacts.
+* Configuration Framework Service \(CFS\) configures nodes using configuration framework. Launches and aggregates the status from one or more Ansible instances against nodes \(node personalization\) or images \(image customization\).
+
+![Boot Nodes](../../img/operations/boot_orchestration/bos_v2_boot.png)
```

Comment:
Step 5 is never shown here. I feel like that should have the Status Operator writing the Component Record.
Step 12 is not shown either. The Session Completion operator should be shown writing the Session Record.

Reply:
I chose not to show steps that included obvious interactions because I thought it muddied the diagram, and that it was more useful to see how the services were connected and what was involved than to show every step.  I'm willing to switch this though if you feel it's confusing.

Reply:
You might make a note to stave off similar protests. An asterisks saying, "Some steps omitted to keep the diagram uncluttered."